### PR TITLE
SCRUM-2223 Fix discrepancy in publication curies used for DA and CR uniqueId fields

### PR DESCRIPTION
**Pull request by markquintontulloch**
Uses AGR reference curie for generating uniqueIds via bulk uploads.

Reload of disease annotations required as DB migration removes all condition relations.

### CODE REVIEW

File: src/main/resources/db/migration/v0.12.0.4__agr_curation_api.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+DELETE FROM diseaseannotation_conditionrelation;
+
+DELETE FROM conditionrelation_experimentalcondition;
+		
+DELETE FROM conditionrelation;
```
Comment (oblodgett) [inline src/main/resources/db/migration/v0.12.0.4__agr_curation_api.sql:5]: @markquintontulloch does this make sense in light of previous migrations?

Comment (markquintontulloch) [reply src/main/resources/db/migration/v0.12.0.4__agr_curation_api.sql:5]: @oblodgett Not sure which previous migrations you mean, but essentially we need to clean up condition relations that have incorrect paper curies in them.  We could just delete any that have 'PMID' or 'AGR-Reference' in the uniqueId but that wouldn't clean up any that were generated in the period between when the LiteratureService started using AGRKB curies, and when we switched to their production site.  That's possibly only an issue on alpha and I think we could clean those up by checking the conditionRelation.singleReference to see if it's obsolete (from previous migration), but given we have to reload all disease annotations anyway (to fix the DA uniqueId) I figured we might as well delete them all.

Comment (oblodgett) [reply src/main/resources/db/migration/v0.12.0.4__agr_curation_api.sql:5]: Right, but taking a bundle of migrations and putting them on beta will everything be ok with the deployment?

Comment (oblodgett) [reply src/main/resources/db/migration/v0.12.0.4__agr_curation_api.sql:5]: Maybe everything will be fine.

**Review by oblodgett — APPROVED** [review]

File: src/main/resources/db/migration/v0.12.0.4__agr_curation_api.sql (hunk 1 above)
Comment (markquintontulloch) [reply src/main/resources/db/migration/v0.12.0.4__agr_curation_api.sql:5]: I don't see why not.  For v0.12 we've got 4 migrations - one to create database indexes, the next to obsolete references, one to cleanup detached notes, and then this one.  Shouldn't be any clashes.

**Pull request merged**